GoogleAnalytics/pages/config.php

```diff
@@ -13,17 +13,41 @@
 
 access_ensure_global_level( ADMINISTRATOR );
```

Comment:
Should we remove the admin access level check and use the manage_plugin_threshold that was referenced below?

Reply:
Done.